script.js

```diff
@@ -284,7 +286,14 @@ const newYouTube = {
     const countString = extractDigitArray.join();
     newYouTube._commentsState.hasGotCount = true;
     newYouTube._commentsState.currentCount = countString;
-    newYouTube._rewriteCommentsCount()
+    
+    const tryRewrite = setInterval(() => {
+      if (newYouTube._rewriteCommentsCount() === false) {
+        newYouTube._rewriteCommentsCount();
+      } else {
+        clearInterval(tryRewrite);
+      }
+    }, 100);
```

Comment:
Fetch of the number of comments may end before the button's injection. So wait for the button.